x-pack/plugins/ml/public/application/jobs/new_job/pages/index_or_search/page.tsx

```diff
@@ -69,6 +72,9 @@ export const Page: FC<PageProps> = ({
                     defaultMessage: 'Saved search',
                   }
                 ),
+                showSavedObject: (savedObject: SavedObject) =>
+                  // ES|QL Based saved searches are not supported across ML, filter them out
```

Comment:
It covers:
* Data Visualizer => Data View
* AIOps Labs
* Anomaly Detection => Jobs